remoteappmanager/static/js/home/controller.js

```diff
@@ -51,5 +51,7 @@ require([
         ga_observer.trigger_application_starting(application.app_data.image.name);
     });
 
+    app_list_view.$on('entry_clicked', function() { app_view.focus_iframe(); });
```

Comment:
I can rename it clicked, but I will still need to emit the event with the `$emit` method. You want me to rename it ?